consutil/main.py

```diff
@@ -38,10 +40,13 @@ def show():
         if lineNum in busyDevices:
             pid, date = busyDevices[lineNum]
             busy = "*"
-        baud = getBaud(lineNum)
+        actBaud, confBaud, _ = getConnectionInfo(lineNum)
+        # repeated "~" will be replaced by spaces - hacky way to align the "/"s
+        baud = "{}/{}{}".format(actBaud, confBaud, "~"*(15-len(confBaud)))
```

Comment:
Will this solution keep the `/` aligned in cases where we have baud rates of varying lengths (i.e. 9600, 115200)?

Reply:
Yes - "~" is multiplied by (15 - len(confBaud)), so the total number of characters to the right of the "/" stays the same

Reply:
The entries of the columns are all right-aligned, so the placement of the "/" only depends on how many chars are to the right of it.
Here's the output I get with varying baud rates:
```
  Line    Actual/Configured Baud    PID                Start Time
------  ------------------------  -----  ------------------------
     0      9600/-
     1    115200/115200
    *2      9600/9600              5718  Thu Aug  9 18:35:35 2018
     3      9600/-
```